datasets/world_bank_intl_debt/series_times/series_times_dag.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from airflow import DAG
+from airflow.contrib.operators import gcs_to_bq, kubernetes_pod_operator
+
+default_args = {
+    "owner": "Google",
+    "depends_on_past": False,
+    "start_date": "2021-03-01",
+}
+
+
+with DAG(
+    dag_id="world_bank_intl_debt.series_times",
+    default_args=default_args,
+    max_active_runs=1,
+    schedule_interval="@daily",
+    catchup=False,
+    default_view="graph",
+) as dag:
+
+    # Run CSV transform within kubernetes pod
+    series_times_transform_csv = kubernetes_pod_operator.KubernetesPodOperator(
+        task_id="series_times_transform_csv",
+        startup_timeout_seconds=600,
+        name="series_times",
+        namespace="default",
+        affinity={
+            "nodeAffinity": {
+                "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                        {
+                            "matchExpressions": [
+                                {
+                                    "key": "cloud.google.com/gke-nodepool",
+                                    "operator": "In",
+                                    "values": ["pool-e2-standard-4"],
+                                }
+                            ]
+                        }
+                    ]
+                }
+            }
+        },
+        image_pull_policy="Always",
+        image="{{ var.json.world_bank_intl_debt.container_registry.run_csv_transform_kub }}",
+        env_vars={
+            "SOURCE_URL": "gs://pdp-feeds-staging/RelayWorldBank/IDS_CSV/IDSSeries-Time.csv",
```

Comment:
I find it odd that the source is a bucket created by ourselves (PDP). Is this CSV file updated by an internal process?

Reply:
As discussed, we are skipping this as of now